sonic-thermalctld/tests/test_thermalctld.py

```diff
@@ -283,3 +284,58 @@ def test_update_thermal_with_exception():
     temperature_updater = TemperatureUpdater(SYSLOG_IDENTIFIER, chassis)
     temperature_updater.update()
     temperature_updater.log_warning.assert_called()
+
+#Modular chassis related tests
```

Comment:
a) Can you please add the UT outcome/logs of thermalCtld updating both local redis-DB and chassis-state-DB with board's temperature sensors? dump of both the DBs
b) Also, would 'show platform temperature' CLI display the temperature sensors? 
Assuming it would pick the data from PMON's local redis-DB

Reply:
```
Output of a thermal sensor in Global-DB

admin@sonic:~$ redis-dump -p 6380 -d 13 -y -k "*Thermal 2*"
 "TEMPERATURE_INFO_16|Thermal 2": {
    "expireat": 1604478797.573126,
    "ttl": -0.001,
    "type": "hash",
    "value": {
      "critical_high_threshold": "N/A",
      "critical_low_threshold": "N/A",
      "high_threshold": "68.0",
      "low_threshold": "0.0",
      "maximum_temperature": "35.0",
      "minimum_temperature": "24.0",
      "temperature": "28.0",
      "timestamp": "20201102 07:49:17",
      "warning_status": "False"
    }
  },

```
```
Output of a thermal sensor in local-DB
admin@sonic:~$redis-dump  -d 6 -y -k "*Thermal 2*"
{
  "TEMPERATURE_INFO|Thermal 2": {
    "expireat": 1604478899.8649862,
    "ttl": -0.001,
    "type": "hash",
    "value": {
      "critical_high_threshold": "N/A",
      "critical_low_threshold": "N/A",
      "high_threshold": "68.0",
      "low_threshold": "0.0",
      "maximum_temperature": "35.0",
      "minimum_temperature": "24.0",
      "temperature": "28.0",
      "timestamp": "20201102 07:49:17",
      "warning_status": "False"
    }
  }
}
```

Reply:
For b), yes 'show platform temperature' would read from local redis-db of sensors on that card, supervisor or line-card.